ui/src/literature/components/Figures/Figure.jsx

```diff
@@ -5,36 +5,39 @@ import Image from 'react-image';
 import { Spin } from 'antd';
 
 import './Figure.scss';
+import Latex from '../../../common/components/Latex';
 
 function renderWithContainer(children) {
-  return (
-    <div className="image-container mh3 ba">
-      {children}
-    </div>
-  )
+  return <figure className="image-container ba">{children}</figure>;
 }
 
 function renderWithoutContainer(children) {
   return children;
 }
 
-function Figure({ url, className, onClick }) {
+const LOADER = (
+  <Spin style={{ margin: 'auto', display: 'block', padding: '2rem 0' }} />
+);
+
+function Figure({ url, className, onClick, caption }) {
   return (
-    <div className="__Figure__">
-        <Image
-          onClick={onClick}
-          className={classNames(className, 'pa1 db center w-auto h-auto', {
-            pointer: onClick,
-          })}
-          src={url}
-          container={renderWithContainer}
-          unloaderContainer={renderWithoutContainer}
-          loader={
-            <Spin
-              style={{ margin: 'auto', display: 'block', padding: '2rem 0' }}
-            />
-          }
-        />
+    <div className="__Figure__ bg-white pa3">
+      <Image
+        onClick={onClick}
+        className={classNames(className, 'pa1 db center w-auto h-auto', {
+          pointer: onClick,
+        })}
+        src={url}
+        container={renderWithContainer}
+        unloaderContainer={renderWithoutContainer}
+        alt="Figure"
+        loader={LOADER}
+      />
+      {caption && (
+        <figcaption>
```

Comment:
In order to make semantic sense we should wrap this in
``<figure>``

Reply:
my bad you got it already

Reply:
but the caption will not be inside I'd change this https://github.com/inspirehep/inspirehep/pull/856/files#diff-1493d7d484f303eaf6e326eca51c36fbR24 into ``figure``

Reply:
Yep, `figcaption` should be inside `figure`.. I thought it was already but apparently not. I'll do that, I will also try to avoid using `react-image` container API, which makes it hard understand how underlying DOM structure of the component.